src/sentry/relay/config/__init__.py

```diff
@@ -300,21 +310,21 @@ class _ConfigBase:
     [1, 2, 3]
     """
 
-    def __init__(self, **kwargs):
-        data = {}
+    def __init__(self, **kwargs: Any) -> None:
+        data: Dict[Any, Any] = {}
```

Comment:
you can use mutablemapping here. doing it this way allows you to control dict mutability via types.

we prefer that pattern in snuba, in sentry i guess we don't have any